tests/arrays_test.fbs

```diff
@@ -0,0 +1,13 @@
+namespace ArraysTest.Test1;
```

Comment:
please also put this in MyGame.Example namespace, so once support for other languages arrives we don't have an explosion of folders.